TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -501,7 +527,12 @@ public void DidChangeConfigurationParams(string[] arguments)
                 Configuration.ExecToStep = ExecutionStep.QualityCheck; //Language Server does not support Cobol Generation for now
 
             var typeCobolOptions = new TypeCobolOptions(Configuration);
-
+#if EUROINFO_RULES
+            ReturnCode optionRetCode = typeCobolOptions.OptionStatusCode;
```

Comment:
Same remark as in CLI, personally I think the log done in `TypeCobolOptions` is enough and that we should not really worry about these kind of exceptions. Checking the file path is enough to ensure a safe load.